ibis/backends/flink/tests/conftest.py

```diff
@@ -42,6 +75,7 @@ class TestConfForStreaming(TestConf):
     def connect(*, tmpdir, worker_id, **kw: Any):
         """Flink backend is created in streaming mode here. To be used
         in the tests under ibis/ibis/backends/flink/tests/.
+        We only use mini cluster here for simplicity.
```

Comment:
Is this true, even if the above code goes the remote cluster route?

Reply:
Yes, `ibis/backends/flink/tests` are not running inside remote cluster since it's using `TestConfForStreaming`